daemon/algod/api/algod.oas3.yml

```diff
@@ -1991,6 +2008,10 @@
             "description": "A boolean option for opting in execution trace features simulation endpoint.",
             "type": "boolean"
           },
+          "scratch-change": {
```

Comment:
I think I like this name ("change") more than ("write"). But either way, we should be consistent.  This option should be "scratch-writes" if we go with "writes".